### PR TITLE
[MOS-528] Fixed BT disconnect option

### PR DESCRIPTION
**Pull request by Lefucjusz**
**Description**

Fix of the issue that after connecting
to Bluetooth device it was impossible
to disconnect using Disconnect button.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [x] Has unit tests if possible.
- [x] Has documentation updated

### CODE REVIEW

**Review by Maciej-Mudita — APPROVED** [review]

File: module-bluetooth/Bluetooth/BluetoothStateMachine.hpp [hunk 1]
```diff
@@ -355,6 +355,7 @@ namespace bluetooth
                         state<Idle> + sml::event<bluetooth::event::VisibilityOn> / HandleSetVisibility = state<Idle>,
                         state<Idle> + sml::event<bluetooth::event::VisibilityOff> / HandleUnsetVisibility = state<Idle>,
                         state<Idle> + sml::event<bluetooth::event::ConnectAudio> / EstablishAudioConnection = state<Idle>,
+                        state<Idle> + sml::event<bluetooth::event::DisconnectAudio> / HandleDisconnect = state<Idle>,
```
Comment (pholat) [inline module-bluetooth/Bluetooth/BluetoothStateMachine.hpp:358]: UT or didn't happen

**Review by pholat — CHANGES_REQUESTED** [review]
UT? :)

File: module-bluetooth/Bluetooth/BluetoothStateMachine.hpp (hunk 1 above)
Comment (SP2FET) [reply module-bluetooth/Bluetooth/BluetoothStateMachine.hpp:358]: +1

**Review by SP2FET — APPROVED** [review]

File: module-bluetooth/Bluetooth/BluetoothStateMachine.hpp (hunk 1 above)
Comment (Lefucjusz) [reply module-bluetooth/Bluetooth/BluetoothStateMachine.hpp]: Added UT and renamed the connect/disconnect events

**Review by SP2FET — APPROVED** [review]

**Review by pholat — APPROVED** [review]

**Pull request merged**